src/social_auth/models.py

```diff
@@ -24,6 +29,57 @@
 CLEAN_USERNAME_REGEX = re.compile(r"[^\w.@+-_]+", re.UNICODE)
 
 
+class HybridUserSocialAuthUtility:
```

Comment:
You're not using the class for much. You could make these module level functions instead 🏌️